source/manual/aws-iam-key-rotation.html.md

```diff
@@ -5,93 +5,131 @@ layout: manual_layout
 type: learn
 section: AWS
 owner_slack: "#govuk-developers"
-last_reviewed_on: 2019-10-31
+last_reviewed_on: 2020-01-17
 review_in: 6 months
 ---
 
 ## Overview
 
-GOV.UK uses [AWS IAM](https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/id_users.html)
-accounts to manage applications access to our AWS infrastructure. The
-[Access Keys](https://docs.aws.amazon.com/en_pv/IAM/latest/UserGuide/id_credentials_access-keys.html)
-associated with these accounts need to be [rotated](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_RotateAccessKey)
-every 3 months.
+GOV.UK uses [AWS IAM] accounts to manage applications access to our AWS
+infrastructure. The [Access Keys] associated with these accounts need to be
+[rotated] every 3 months.
 
-## How to rotate Access Keys
+Rotating AWS IAM keys requires permissions which are available with the `admin`,
+`internal-admin`, `platformhealth-poweruser` or `poweruser` roles. If you can't
+assume one of these roles, you won't be able to rotate keys.
 
-> **Note**
->
-> Rotating AWS IAM keys requires permissions which are available with the
-> `admin`, `internal-admin`, `platformhealth-poweruser` or `poweruser` roles.
-> If you can't assume one of these roles, you won't be able to rotate keys.
+## How to rotate Access Keys
 
-Log in to the [IAM Management Console](https://console.aws.amazon.com/iam/home?region=eu-west-1#/users)
-for the environment you want to rotate keys for and you'll see a list of users
-and the age of their access keys.
+Log in to the [IAM Management Console] in the environment you want to rotate
+keys for and you'll see a list of users and the age of their access keys.
 
 ### 1. Find users access key
 
-Click on the user whose key you want to rotate. Click on `Security Credentials`. Make a note of the Access Key ID.
+- Click on the user whose key you want to rotate.
+- Click on the `Security Credentials` tab.
 
-### 2. Find where the key is used
+Users should always only have one access key. The only time a user should have
+two keys is when we are in the process of rotating the old one. For this reason,
```

Comment:
I'd perhaps say "rotating the key that is in use", rather than "rotating the old one", at least that makes more sense to me.